openassessment/xblock/leaderboard_mixin.py

```diff
@@ -76,16 +76,18 @@ def render_leaderboard_complete(self, student_item_dict):
         for score in scores:
             score['files'] = []
             if 'file_keys' in score['content']:
-                for key in score['content']['file_keys']:
-                    url = ''
-                    try:
-                        url = file_upload_api.get_download_url(key)
-                    except FileUploadError:
-                        pass
-                    if url:
-                        score['files'].append(url)
+                file_keys = score['content'].get('file_keys', [])
+                descriptions = score['content'].get('files_descriptions', [])
+                for idx, key in enumerate(file_keys):
```

Comment:
@noraiz-anwar Let's keep it simple as that, I find it more readable and understandable. Although your suggestion has less LOC but I will vote for readability.

What do you think?